cypress/page/personal-form.page.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable require-jsdoc */
+class PersonalFormPage {
+  private firstName: string;
+  private lastName: string;
+  private email: string;
+  private gender: string;
+  private birthDate: string;
+  private mobile: string;
+  private hobbies: string;
+  private currentAddress: string;
+  private state: string;
+  private city: string;
+  private submitBtn: string;
+  private personalFormPageURL: string;
+  private modalTitle: string;
+
+  constructor() {
+    this.firstName = "#firstName";
+    this.lastName = "#lastName";
+    this.email = "#userEmail";
+    this.gender = "#genterWrapper";
+    this.mobile = "#userNumber";
+    this.birthDate = "#dateOfBirthInput";
+    this.hobbies = "#hobbiesWrapper";
+    this.currentAddress = "#currentAddress";
+    this.state = "#react-select-3-input";
+    this.city = "#react-select-4-input";
+    this.submitBtn = "#submit";
+    this.personalFormPageURL = "https://demoqa.com/automation-practice-form";
+    this.modalTitle = "#example-modal-sizes-title-lg";
+  }
+
+  public visitFormPage(): void {
+    cy.visit(this.personalFormPageURL);
+  }
+
+  public fillForm(userInfo:
+    {name: string,
+        lastName: string,
+        email: string,
+        gender: string,
+        mobileNumber: string,
+        dateOfBirth: string,
+        hobbies: string[],
+        currentAddress: string,
+        state: string,
+        city: string
+       }, assertMsg: string): void {
```

Comment:
can you define an interface or type instead of define the anonymous class here. See this link for more info: https://www.typescriptlang.org/docs/handbook/2/objects.html

Reply:
thank you for the info, I'll use the interface!